.travis.yml

```diff
@@ -8,4 +8,6 @@ install:
 script:
 - tox
 after_success:
-- coveralls
\ No newline at end of file
+- coveralls
+notifications:
+  email: false
```

Comment:
Now TravisCI it sends a notification for every new branch. Since we usually check it in GitHub, I don't find useful email notifications. However, if you find it useful I can keep them :smile:

Reply:
I prefer to keep them